packages/apps/src/themes.ts

```diff
@@ -26,10 +28,14 @@ export const darkTheme: ThemeDef = {
   bgToggle: '#58595f',
   borderTable: 'transparent',
   borderTabs: 'transparent',
+  bountyBGTableOddRow: '#111218',
+  bountyBorderTable: '#2F313C',
+  bountyColor: '#eeeeee',
```

Comment:
Note sure about these here at all - since they are only bounty-specific and won't be used anywhere so doesn't have applicability in the globals.

I suppose the right pattern in bounties would therefore be `${theme == 'dark' ? '#111218' : '#fff'}`

This localizes it to bounties where it departs from the standard. (If there is every more use it can be extracted into the global)

Reply:
Looks all good with the recent changes.